src/conn/conn_api.c

```diff
@@ -1834,12 +1834,6 @@ __wt_debug_mode_config(WT_SESSION_IMPL *session, const char *cfg[])
     return (0);
 }
 
-/* Simple structure for name and flag configuration searches. */
```

Comment:
I moved this somewhere shared, so that I could pull out verbose options for the data handle as well.